CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
+## [4.0.0] - 2022-03-31
+
+### Changed
+
+- Update the list-with-actions component and added the ability to add custom classes to the row element
+- Updated hmrc-frontend to 5.0.1
```

Comment:
Looking at the `LibDependencies`, I think this is actually `5.0.2`, not `5.0.1`.